tests/src/app/testqgsmaptoollabel.cpp

```diff
@@ -298,10 +298,8 @@ class TestQgsMapToolLabel : public QObject
       tool->mCurrentLabel = QgsMapToolLabel::LabelDetails( pos, canvas.get() );
 
       // defaults to bottom left
-      QString hali, vali;
-      tool->currentAlignment( hali, vali );
-      QCOMPARE( hali, QStringLiteral( "Left" ) );
-      QCOMPARE( vali, QStringLiteral( "Bottom" ) );
+      QgsMapToolLabel::LabelAlignment labelAlignment = tool->currentAlignment();
+      QCOMPARE( labelAlignment, QgsMapToolLabel::LabelAlignment::HalfCenter );
```

Comment:
Can you explain why this result has changed from bottom left to half center?

Reply:
Yes, this is because the former implementation was just ignoring the `quadOffset` property as of line 271. I should change the comment anyway.
Effectively if I pythonize this code and execute in QGIS the label at the beginning is aligned half center and not bottom left.